### PR TITLE
Dalamud Flytext functionality

### PR DESCRIPTION
**Pull request by lmcintyre**

### CODE REVIEW

**Comment by rlynch-ironnet** [comment]
Do me a favor and put it in `Dalamud.Game.Gui.FlyText`

File: Dalamud/Game/Text/SeStringHandling/SeStringManager.cs [hunk 1]
```diff
@@ -47,6 +49,24 @@ public SeString Parse(byte[] bytes)
             return new SeString(payloads);
         }
 
+        /// <summary>
```
Comment (daemitus) [inline Dalamud/Game/Text/SeStringHandling/SeStringManager.cs:52]: Do we need this compared to MemoryHelper.ReadSeString?

File: Dalamud/Game/Gui/FlyTextGui.cs [hunk 2]
```diff
@@ -0,0 +1,312 @@
...
+using System.Diagnostics;
+using System.Runtime.InteropServices;
+using System.Threading.Tasks;
+
+using Dalamud.Game.Text.SeStringHandling;
+using Dalamud.Hooking;
+using FFXIVClientStructs.FFXIV.Client.UI;
+using Serilog;
+
+namespace Dalamud.Game.Gui
```
Comment (daemitus) [inline Dalamud/Game/Gui/FlyTextGui.cs:11]: I meant add another `FlyText` to the namespace, something like that anyways. Anything past 2 files gets a namespace. `Dalamud.Game.Gui.FlyText.FlyTextGui`

**Review by daemitus — COMMENTED** [review]
@goaaats looks good, besides my few comments/questions.

File: Dalamud/Game/Gui/FlyTextGui.cs (hunk 2 above)
Comment (lmcintyre) [reply Dalamud/Game/Gui/FlyTextGui.cs]: Moved 🙂

**Pull request merged**